tests/test_validation.py

```diff
@@ -1131,9 +1137,9 @@ def step_1(self):
             if pd.isnull(temp) and pd.isnull(self.dataframe[keywords][0]):
                 self.assertTrue(pd.isnull(self.dataframe[keywords][0]))
             elif keywords == 'All Threshold Crossing Times':
-                self.assertEqual(self.dataframe[keywords][0], str([temp]))
+                self.assertAlmostEqual(self.dataframe[keywords][0], str([temp]))
```

Comment:
This looks like `assertAlmostEqual()` is being passed a `str`.  That can't work, can it?